.pre-commit-config.yaml

```
@@ -0,0 +1,26 @@
+repos:
+
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+    - id: trailing-whitespace
+    - id: check-added-large-files
+    - id: mixed-line-ending
+      args: ['--fix=lf']
+
+-   repo: https://github.com/pre-commit/mirrors-isort
+    rev: v5.8.0
+    hooks:
+    -   id: isort
+        args: ['--multi-line=3', '--trailing-comma', '--force-grid-wrap=0', '--use-parentheses', '--line-width=88']
+
+-   repo: https://github.com/humitos/mirrors-autoflake.git
+    rev: v1.1
+    hooks:
+    -   id: autoflake
+        args: ['--in-place', '--remove-all-unused-imports']
```

```suggestion
  - repo: https://github.com/myint/autoflake.git
    rev: v1.4
    hooks:
    -   id: autoflake
        args: ['--in-place', '--remove-all-unused-imports', '--ignore-init-module-imports']
```

This is the actual autoflake repo, not a mirror. Plus with the `--ignore-init-module-imports` argument, all `# F401` comments could be removed from init files.